openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -272,6 +275,12 @@ def _submit_deadline_post_job(self, instance, job, instances):
 
         priority = self.deadline_priority or instance.data.get("priority", 50)
 
+        instance_settings = self.get_attr_values_from_data(instance.data)
+        initial_status = instance_settings.get("publishJobState", "Active")
+        # TODO: Remove this backwards compatibility of `suspend_publish`
```

Comment:
why remove?

Reply:
The logic has moved since now it is an attribute definition, so now it has a clear toggle in the Publisher UI.